src/Illuminate/Container/Container.php

```diff
@@ -1181,7 +1181,7 @@ public static function getInstance()
      * Set the shared instance of the container.
      *
      * @param  \Illuminate\Contracts\Container\Container|null  $container
-     * @return static
+     * @return static|ContainerContract
```

Comment:
Shouldn't it be fully qualified class name here?

Reply:
@a-komarev , I`m not sure. Because ContainerContract already used in this file.

Reply:
Yes, but it's an alias, not a FQCN, but maybe it's just my vision of the code styling.

Reply:
It should be FQCN like in method's parameter.

Reply:
In method it is a 
```PHP
ContainerContract $container = null
```

Should I really change it to the 
```PHP
Illuminate\Contracts\Container\Container
```
in Doc?

Reply:
Changed